pkg/features/features.go

```diff
@@ -28,7 +28,7 @@ const (
 
 	// Logging enables logging stack for clusters.
 	// owner @mvladev,@vlpanov
-	// alpha: v0.13.0
+	// alpha: v0.18.0
```

Comment:
This comment marks since which version the feature is `alpha` - hence revert it to `v0.13.0`. 

However, I would appreciate if you update the owners - I think that you can take the ownership of the `Logging` feature.